src/styles/tools/_variebles.scss

```diff
@@ -0,0 +1,10 @@
+
+$light-gray: #616070;
+$black: #060b35;
+$light-blue: #00acdc;
+$font-size-medium: 12px;
+$star-size: 16px;
+$card-width: 200px;
+$right-gap: 50px;
```

Comment:
when you have gap in flex, do you need margin on the elements (look on value that I mantioned = 48px)